README.md

```diff
@@ -289,6 +293,7 @@ Available types:
 - `isVoid` <- whether type is Void (`Void` or `()`)
 - `isTuple` <- whether given type is a tuple
 - `tuple.elements` <- if given type is a tuple returns its elements information (*TupleType.Element*)
+- `attributes` <- type attributes, i.e. `typeName.attributes.escaping`, or `typeName.escaping`
```

Comment:
I think we should always allow only accessing those via `attributes` rather than doing the trick of `variable.escaping`, it's not a big deal to write it in the template and it's more consistent and avoids potential clashes